src/haz3lcore/dynamics/Builtins.re

```diff
@@ -33,7 +33,27 @@ module Pervasives = {
   module Impls = {
     open EvaluatorMonad;
     open EvaluatorResult;
-
+    /* int_of_string implementation. */
+    let int_of_string = (name, r1) =>
+      switch (r1) {
+      | BoxedValue(StringLit(f) as d1) =>
+        let f = Re.Str.string_after(f, 1);
+        let f = Re.Str.string_before(f, String.length(f) - 1);
+        if (String.length(f) == 0) {
```

Comment:
instead of these checks, use ocaml's int_of_string_opt (see float-polish for an example)

Reply:
Got it and it has already been fixed in float-polish branch. 
Now the float-polish branch is merged. :)